cluster/ci/config/prow/plugins.yaml

```diff
@@ -343,6 +343,7 @@ plugins:
   - trigger
 
   openshift/installer:
+  - approve
   - docs-no-retest
```

Comment:
`docs-no-retest` can be removed in a follow-up; it's redudant now that we use tide.

/lgtm